src/_damon.py

```diff
@@ -1525,11 +1525,15 @@ def commit_quota_goals(kdamond_idxs):
         return 'debugfs interface does not support commit_quota_goals()'
     return _damon_fs.commit_quota_goals(kdamond_idxs)
 
-def commit(kdamonds, commit_quota_goals_only=False):
-    if not commit_quota_goals_only:
+def commit(kdamonds, commit_quota_goals_only=False, commit_targets_only=False):
+    if not commit_quota_goals_only and not commit_targets_only:
         err = stage_kdamonds(kdamonds)
         if err:
             return 'staging updates failed (%s)' % err
+    elif commit_targets_only:
+        err = _damon_fs.stage_kdamonds_targets(kdamonds)
```

Comment:
If `_damon_fs` is `_damon_dbgfs`, this will crash.  Let's add _damon_dbgfs case handling, similar to that on `commit_quota_goals()` function.

Reply:
Gotcha. I'll try to get that done soon.